sdk_container/src/third_party/coreos-overlay/app-admin/sdnotify-proxy/sdnotify-proxy-9999.ebuild

```
@@ -2,17 +2,15 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-CROS_WORKON_PROJECT="kinvolk/sdnotify-proxy"
-CROS_WORKON_LOCALNAME="sdnotify-proxy"
-CROS_WORKON_REPO="https://github.com"
+EGIT_REPO_URI="https://github.com/kinvolk/sdnotify-proxy.git"
 COREOS_GO_PACKAGE="github.com/coreos/sdnotify-proxy"
```

While at it, I'd change this to flatcar too - the package name is not mentioned anywhere:
```suggestion
COREOS_GO_PACKAGE="github.com/flatcar/sdnotify-proxy"
```

---

The string gets baked into the binary, but that's fine. Only the last part is used to name the binary. Maybe the rest is more significant in libraries, but I don't know much about Go.